webapp/src/css/dropdown.less

```diff
@@ -109,19 +112,6 @@
       > a[role="menuitem"] {
         padding: 5px 20px 3px 35px;
         text-indent: -20px;
-
-        &.indent-1 {
```

Comment:
This isn't scalable, there are projects with more than 6 levels, the [above code](https://github.com/medic/cht-core/pull/7696/files#diff-0b56f98a5ceb46296cd8b10467780cfa4349edf0fdf2171bd6b4e9f2841a45cdR100) fixes that issue.